shell/browser/feature_list.cc

```diff
@@ -46,6 +47,12 @@ void InitializeFeatureList() {
       // 'custom dictionary word list API' spec to crash.
       std::string(",") + spellcheck::kWinDelaySpellcheckServiceInit.name;
 #endif
+
+#if BUILDFLAG(IS_MAC)
+  // Disable window occlusion checker.
+  disable_features +=
+      std::string(",") + features::kMacWebContentsOcclusion.name;
```

Comment:
Current occlusion tracker does not work for hidden startups, since detection only happens for move, resize or fullscreen signals. Tried another approach but doesn't work for all of them. Tests are not reliable for me locally even for the legacy path so will leave this disabled for now, 

```
With the new occlusion detection enabled via
https://chromium-review.googlesource.com/c/chromium/src/+/5605407
starting the window as hidden previously resulted in
Visibility::kOccluded (via viewDidMoveToWindow) -> Visibility::kHidden (via viewDidHide)
but it now starts with Visibility::kVisible -> Visibility::kHidden
which throws off the initial document.visibilityState checks.

When starting as hidden WebContentsOcclusionCheckerMac will not have
performed any occlusion checks, which results in the initial visibility
state to be visible. This makes sense for chrome since they always start
as visible, but we require support for starting as hidden. Current fix
adds a new property to the occlusion checker that confirms whether a check
has been performed, if yes then we rely on the results from the checker
if not we consider this is the initial state and rely on NSWindow occlusionState
to decide the view to be visible or occluded.
```

Reply:
@MarshallOfSound since you have touched this codepath the most, PTAL incase you have cycles. Thanks!